src/backend/TrafficCourts/Workflow.Service/Consumers/SearchDisputeConsumer.cs

```diff
@@ -37,13 +37,20 @@ public async Task Consume(ConsumeContext<SearchDisputeRequest> context)
                 if (searchResult is null || searchResult.Count == 0)
                 {
                     _logger.LogError("Dispute not found");
+                    await context.RespondAsync<SearchDisputeResponse>(new SearchDisputeResponse());
                 }
                 DisputeResult? result = searchResult?.OrderByDescending(d => d.DisputeId).FirstOrDefault();
+
+                // TODO: get value from DisputeResult
+                ICollection<JJDispute> jJDisputes = await _oracleDataApiService.GetJJDisputesAsync("", ticketNumber, issuedTime, context.CancellationToken);
+                JJDisputeHearingType? hearingType = jJDisputes.Select(i => i.HearingType).FirstOrDefault();
+
                 await context.RespondAsync<SearchDisputeResponse>(new SearchDisputeResponse()
                 {
-                    DisputeId = result?.DisputeId.ToString(),
+                    NoticeOfDisputeGuid = result?.DisputeId.ToString(), // TODO: change this to NoticeOfDisputeGuid
                     DisputeStatus = result?.DisputeStatus.ToString(),
-                    JJDisputeStatus = result?.JjDisputeStatus?.ToString()
+                    JJDisputeStatus = result?.JjDisputeStatus?.ToString(),
+                    HearingType = hearingType?.ToString()

```

Comment:
I think I see what you are doing here. Since hearingType is not yet in the DisputeResult returned object, you are fetching the value via the GetJJDisputesAsync call.  But when the oracle-data-api is updated lines 44-46 will go away, correct?

Reply:
Yes

Reply:
Okay, great.